package.json

```diff
@@ -9,8 +9,9 @@
     "lib"
   ],
   "dependencies": {
+    "babel-plugin-syntax-object-rest-spread": "^6.13.0",
     "find-up": "^2.1.0",
-    "istanbul-lib-instrument": "^1.7.5",
+    "istanbul-lib-instrument": "^1.8.0",
```

Comment:
Explicitly bumping to ensure https://github.com/istanbuljs/istanbuljs/pull/82 is included